src/content/language/patterns.md~

```diff
@@ -0,0 +1,447 @@
+---
+title: Patterns
```

Comment:
Did you intend to make a copy of this file with a `~` at the end? Could you double check that your changes to this file are showing up correctly in the diff?